tests_e2e/orchestrator/runbook.yml

```diff
@@ -54,6 +54,8 @@ variable:
       - publish_hostname
       - recover_network_interface
       - cgroup_v2_disabled
+      - extensions_policy
+      - extensions_policy_disabled
```

Comment:
rather that "disabled", should be "unsupported" (the "disabled" test enables the feature)